crates/oxc_linter/src/snapshots/require_to_throw_message.snap

```diff
@@ -0,0 +1,37 @@
+---
+source: crates/oxc_linter/src/tester.rs
+expression: require_to_throw_message
+---
+
+  ⚠ eslint-plugin-jest(require-to-throw-message): Require a message for `toThrow()`.
+   ╭─[require_to_throw_message.tsx:1:41]
+ 1 │ expect(() => { throw new Error('a'); }).toThrow();
+   ·                                         ───────
+   ╰────
+  help: Add an error message to "toThrow"()
+
+  ⚠ eslint-plugin-jest(require-to-throw-message): Require a message for `toThrow()`.
+   ╭─[require_to_throw_message.tsx:1:41]
+ 1 │ expect(() => { throw new Error('a'); }).toThrowError();
+   ·                                         ────────────
+   ╰────
+  help: Add an error message to "toThrowError"()
```

Comment:
Is this mean `"toThrowError()"` or `"toThrowError"` ?

Reply:
Sorry, it was a mistake.
I have modified it to display the matcher related to errors.
c926297d16117121a3240f61a2b16edbf83cc51c

Reply:
I removed the `()` because it was unnecessary.
b08bf7ac961146f0d41e08c42fa8121fb249a5fc